src/rpcblockchain.cpp

```diff
@@ -665,6 +666,17 @@ UniValue getblockchaininfo(const UniValue& params, bool fHelp)
             "        },\n"
             "        \"reject\": { ... }      (object) progress toward rejecting pre-softfork blocks (same fields as \"enforce\")\n"
             "     }, ...\n"
+            "  ],\n"
+            "  \"upgrades\": [                (array) status of upgrades\n"
+            "     {\n"
+            "        \"upgradeid\": \"xxxx\",    (string) id of upgrade\n"
+            "        \"name\": \"xxxx\",         (string) name of upgrade\n"
+            "        \"status\": \"xxxx\",       (string) status of upgrade\n"
+            "        \"activationheight\": \"xxxxxx\",  (numeric) block height of activation\n"
+            "        \"txembargostart\": \"xxxxxx\",    (numeric) block height that transaction embargo begins\n"
+            "        \"txembargoend\": \"xxxxxx\",      (numeric) block height that transaction embargo ends\n"
```

Comment:
We aren't having a transaction embargo any more - at least, not one that is consensus-enforced.

We might still want this here to indicate any window that the local node will prevent transactions from being sent. However, we should add that in a PR that implements such a feature.